projects/demo-integrations/cypress/tests/kit/time/time-mode.cy.ts

```diff
@@ -345,5 +345,65 @@ describe('Time', () => {
                 });
             });
         });
+
+        describe('HH', () => {
+            describe('default segments', () => {
+                beforeEach(() => {
+                    cy.visit(`/${DemoPath.Time}/API?mode=HH`);
+                    cy.get('#demo-content input')
+                        .should('be.visible')
+                        .first()
+                        .focus()
+                        .clear()
+                        .type('{moveToStart}')
```

Comment:
I think we can drop this line because caret index is already `0` after `.clear()`